src/qt/locale/bitcoin_fr.ts

```diff
@@ -1495,15 +1495,15 @@
     </message>
     <message>
         <source>Never share your private key with anyone</source>
-        <translation type="unfinished"></translation>
+        <translation>Ne partagez jamais votre clé privée avec quiconque</translation>
     </message>
     <message>
         <source>Who owns the private keys, owns the coins</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <source>To see ongoing development and contribute, check out the Dogecoin Core repository on GitHub</source>
-        <translation type="unfinished"></translation>
+        <translation>Pour suivre le développement en cours et contribuer, consultez le repo Dogecoin Core sur GitHub.</translation>
```

Comment:
Corriger "repo" du mot anglais repository par "dépôt" qui est plus correct en français, "Pour suivre le développement en cours et contribuer, consultez le dépôt Dogecoin Core sur GitHub."

Reply:
Line 1502 can also be completed with "Celui qui possède les clés privées, possède les pièces"